gatsby-ssr.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/ssr-apis/
+ */
+
+// You can delete this file if you're not using it
```

Comment:
Especially this one..

Reply:
Yeah I think it's safe to delete this now, we are not going to use it any time soon here.